### ISSUE TITLE
[BUG] At a glance not disappearing when turned off

### ISSUE THREAD

**Issue by elisenlebkuch** [Deletescape-Media/Lawnchair#5103]
### Describe the bug

When I updated lawnchair today, the "at a glance" widget appeared dispite it being turned off. Toggling it on and off did nothing, too. I tried reinstalling but it also wouldn't work then. I don't know exactly which nightly version it began to appear on, but it was one released today.

### Steps to reproduce

1. Install/update lawnchair
2. try to turn off "at a glance"
3. look at the home screen

### Expected behavior

It should be disappearing when turning it off.

### Device information

- Device: OnePlus 12
- OS: YAP latest
- App version: some nightly released on dec 20 2024

**Comment by bappitybup** [comment]
can confirm from version 4a5f397

**Comment by Wierius799** [comment]
Same here, it started with the nightly that was released right before this one too.
Pixel 7 Pro, Android 15.

Edit: I must add that the same update that introduced this behavior, also fixed an issue where shortcuts and widgets were pushed off screen (whenever I changed the layout) into a hidden screen page, the update added this "At a glance" widget into that hidden page, so it's now shown and I was able to remove the duplicated shortcuts and widgets.

Additionally, the new shortcuts of recently installed app weren't appearing in my homescreen and it seems to have been related to that too, as all of those new shortcuts were also being created in the hidden screen and I was able to recover them.

I'm awaiting for drawerless mode, I've been using the workaround of disabling the open drawer gesture but the issue with the shortcuts made me turn it back on, sadly. (At least that seems to be sorted out)
